File: CSharpFundamentals/CSharpOOPbasics/InterfacesAndAbstractionEx/BorderControl/Citizen.cs
namespace BorderControl
{
    public class Citizen : ITraceble
    {
        public string Name { get; private set; }
        public int Age { get; private set; }
        public string Id { get; private set; }

        public Citizen(string name, int age, string id)
        {
            Name = name;
            Age = age;
            Id = id;
        }
    }
}

File: JavaFundamentals/JavaOOP/Interfaces-And-Abstraction/src/border_control/Citizen.java
package border_control;

public class Citizen implements Identifiable {
    private String id;
    private String name;
    private int age;

    public Citizen(String name, int age, String id) {
        this.id = id;
        this.name = name;
        this.age = age;
    }

    public String getName() {
        return this.name;
    }

    public int getAge() {
        return this.age;
    }

    @Override
    public String getId() {
        return this.id;
    }
}
